src/leap/mx/mail_receiver.py

```diff
@@ -75,6 +76,11 @@ class MailReceiver(Service):
     """
     RETRY_DIR_WATCH_DELAY = 60 * 5  # 5 minutes
 
+    """
+    Number of days to keep stalled emails
+    """
+    BOUNCE_TIMEOUT_DAYS = 5
```

Comment:
Not sure if I agree with that, as there is other constants in this class like `RETRY_DIR_WATCH_DELAY` and `PROCESS_SKIPPED_INTERVAL`. But if you strongly disagree I'll happy to change it.